content/departments/product-engineering/engineering/cloud/devops/managed/creation_process.md

```diff
@@ -43,23 +45,24 @@ For basic operations like accessing an instance for these steps, see [managed in
    - Email: `managed+$COMPANY@sourcegraph.com` (note `+` sign not `-`)
    - Username: `sourcegraph-admin`
    - Password: Use the password previously created and stored in 1password.
-1. Create a token for the account under `/users/sourcegraph-admin/settings/tokens` called `managed-instances` and add it as "token" under the 1password entry for the admin account.
-1. Navigate to Grafana and confirm the instance looks healthy.
+1. Create a token for the account under `/users/sourcegraph-admin/settings/tokens` called `managed-instances` with `site-admin:sudo` checked and add it as "token" under the 1password entry for the admin account.
```

Comment:
I don't think that `site-admin:sudo` is required, just the normal `user:all` (the default). This token is only used for posting update banners so it doesn't need permission to impersonate.

Reply:
Ah, we weren't sure about that one. Thx for clarifying :)

Reply:
+1 on this ,  just the `normal user:all`  is need